apps/explorer/src/app/components/txs/tx-order-type.tsx

```diff
@@ -46,7 +46,7 @@ const displayString: StringMap = {
   'Batch Market Instructions': 'Batch',
   'Stop Orders Submission': 'Stop',
   StopOrdersSubmission: 'Stop',
-  StopOrdersCancellation: 'Cancel stop',
+  StopOrdersCancellation: 'Cancel stop order',
```

Comment:
```suggestion
  StopOrdersCancellation: 'Cancel stop',
```

I prefer to keep these shorter. But this could go either way.